_posts/2015-05-23-photoacoustic-penetration.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Hearing with light: photoacoustic radiology improved"
```

Comment:
You don't talk about "hearing" beyond the first paragraph definition of PA. In the end it seems to be more about seeing than hearing